enable-tls-between-components.md

```diff
@@ -49,9 +49,12 @@ Currently, it is not supported to only enable encrypted transmission of some spe
 
         ```toml
         [security]
-        # set the path for certificates. Empty string means disabling secure connections.
+        ## The path for certificates. Empty string means that secure connections are disabled.
+        # Path of the file that contains list of trusted SSL CAs. If set, the following settings `cert_path` and `key_path` are also needed.
```

Comment:
@lysu PTAL the description changes in this file, thanks~

Reply:
LGTM for those description